extensions/arc/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "arc",
+  "displayName": "Azure Arc",
```

Comment:
Let's make this a localized string (given that it's a product name it probably won't be any different but keeps it consistent)